features/schema.py

```diff
@@ -195,6 +195,7 @@ class Meta:
     ahti_id = graphene.String(required=True)
     source = graphene.Field(FeatureSource, required=True)
     name = graphene.String(required=True)
+    one_liner = graphene.String()
```

Comment:
Since the value will always be at least and empty string i.e. `""`, you could also make this `graphene.String(required=True)`

Reply:
Should we also consider here if we want to turn the description field into a description type in the API?

```
"Description of a Feature"
type Description {
    "Introductory text"
    intro: String
    "Description content"
    content: DescriptionContent
}
```

Reply:
You could add this field to the api related tests.

Reply:
That's another thing we discussed privately. I would be okay with not adding the complexity of refactoring "description" for now, since I don't think we are going to be changing the description further to a well-defined content structure anytime soon.

Reply:
That sounds reasonable.